dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -142,6 +143,97 @@ static ClassDumper create(List<Path> jarFilePaths) {
     return methodReferences.build();
   }
 
+  static SymbolTable scanClassSymbolTable(JavaClass javaClass) {
+    SymbolTable.Builder symbolTableBuilder = SymbolTable.builder();
+    ImmutableSet.Builder<MethodSymbolReference> methodReferences =
+        symbolTableBuilder.methodReferencesBuilder();
+    ImmutableSet.Builder<FieldSymbolReference> fieldReferences =
+        symbolTableBuilder.fieldReferencesBuilder();
+
+    // TODO(suztomo): Read class references (inheritance) from javaClass file
+    ImmutableSet.Builder<ClassSymbolReference> classReferences =
+        symbolTableBuilder.classReferencesBuilder();
+
+    String sourceClassName = javaClass.getClassName();
+    ConstantPool constantPool = javaClass.getConstantPool();
+    Constant[] constants = constantPool.getConstantPool();
+    List<Constant> methodrefConstants = Arrays.stream(constants)
+        .filter(Predicates.notNull())
+        .collect(Collectors.toList());
```

Comment:
What about this? Then you don't need the `methodRefConstants` variable at all, and you don't need to create a list object.

```suggestion
        .forEach(constant -> { ... });
```